client/src/components/dashboard/NotationDisplay.js

```diff
@@ -2,117 +2,117 @@ import React, { useState } from 'react';
-    const notation = 'insert notation info here';
-    const [tooltipClicked, setTooltipClicked] = useState('Copy Text');
+  const classes = useStyles();
+  const { open, handleClose, name, imageSrc, notation } = props;
+  const [tooltipClicked, setTooltipClicked] = useState('Copy Text');
 
-    const closeTooltip = () => {
-        setTooltipClicked('Copy Text');
-    }
+  const closeTooltip = () => {
+    setTooltipClicked('Copy Text');
+  }
 
-    const clickTooltip = () => {
-        setTooltipClicked('Copied!')
-        navigator.clipboard.writeText(notation);
-    }
+  const clickTooltip = () => {
+    setTooltipClicked('Copied!')
+    navigator.clipboard.writeText(notation);
+  }
 
-    return (
-        <Box className={classes.root}>
-            <Dialog className={classes.dialog} onClose={handleClose} open={open}>
-                <DialogTitle className={classes.title}>
-                    You Chose To Play: 
-                    <h1 className={classes.name}>{name}</h1>
-                    <IconButton className={classes.closeButton} onClick={handleClose}>
-                        <CloseIcon />
-                    </IconButton>
-                </DialogTitle>
-                <DialogContent dividers>
-                    <img className={classes.image} src={imageLink} alt={name} />
-                    <Box className={classes.notationCopy}>
-                        <TextField 
-                            label='Notation Info'
-                            id='outlined-read-only-input'
-                            variant='outlined' 
-                            defaultValue={notation} 
-                            fullWidth 
-                            InputProps={{
-                                readOnly: true,
-                            }}
-                        />
-                        <Tooltip 
-                            className={classes.tooltip} 
-                            onClose={closeTooltip} 
-                            title={tooltipClicked} 
-                            placement='top' 
-                            arrow 
-                            interactive
-                        >
-                            <IconButton className={classes.copyButton} onClick={clickTooltip}>
-                                <FileCopyIcon />
-                            </IconButton>
-                        </Tooltip>
-                    </Box>
-                    <Box>
-                        <Button className={classes.buttons} variant='outlined'>Play</Button>
-                        <Button className={classes.buttons} variant='outlined' onClick={handleClose}>Cancel</Button>
-                    </Box>    
-                </DialogContent>
-            </Dialog>
+  return (
+    <Dialog className={classes.dialog} onClose={handleClose} open={open}>
+      <DialogTitle className={classes.title}>
+        You Chose To Play
+        <Typography className={classes.name}>
+          {name}
```

Comment:
super small thing but can we bold this? 

![image](https://user-images.githubusercontent.com/24284033/122652352-4f3d9b80-d0f3-11eb-9f51-3c99cd8d5d8a.png)

Reply:
Good idea!